server/routes/shared_api/autocomplete/autocomplete.py

```diff
@@ -51,6 +52,7 @@ def autocomplete():
   place_id_to_dcid = []
   if place_ids:
     place_id_to_dcid = json.loads(findplacedcid(place_ids).data)
+  logging.info("Found %d place ID to DCID mappings.", len(place_id_to_dcid))
```

Comment:
should we also add a keyword like "place_autocomplete" or something at the beginning of each logging message so we can filter logs

Reply:
done.